%YAML:1.0
---
# Date: 2022-04-29 02:36:37.092549
camera_pos: !!opencv-matrix
   rows: 3
   cols: 1
   dt: d
   data: [ 5.2874959935565036e-01, 1.4109928684257684e+00,
       1.0017304171530266e+00 ]
static_rvec: !!opencv-matrix
   rows: 3
   cols: 1
   dt: d
   data: [ 9.5564411818394729e-03, 2.7921588217893976e+00,
       -1.2072654020587834e+00 ]
static_tvec: !!opencv-matrix
   rows: 3
   cols: 1
   dt: d
   data: [ 3.7338829107478239e-01, -2.2146231327793886e-01,
       1.7565524387568257e+00 ]
mean: !!opencv-matrix
   rows: 3
   cols: 2
   dt: d
   data: [ 39., 94., 516., 214., 120., 258. ]
std: !!opencv-matrix
   rows: 3
   cols: 2
   dt: d
   data: [ 0., 0., 0., 0., 0., 0. ]
